libs/features/mas/src/settings.js

```diff
@@ -49,6 +49,8 @@ function getSettings(config = {}, service) {
     // TODO: add alias names for meta, search and storage
     // See https://git.corp.adobe.com/wcms/tacocat.js/pull/348#discussion_r6557570
     const { commerce = {} } = config;
+    if (window.location.pathname.includes('/tools/ost')) commerce.allowOverride = 'true';
```

Comment:
MAS feature should not know about Milo OST.

`allow-override` is already enabled on Milo stage via `merch.js`.

for OST, I recommend to initialize commerce using `initService` in ost.js 
currently OST initializes `mas-commerce-service` if it doesn't find one already, so we need to make sure that `mas-commerce-service` is initialized by `merch.js`

Reply:
I spent hours before submitting this PR to figure out
1. why [allow-overide in merch.js](https://github.com/adobecom/milo/blob/stage/libs/blocks/merch/merch.js#L988) isn't working for /tools/ost page.
2. The page itself has `mas-commerce-service` but I failed to figure out how to add `allow-overide` attribute into it.
<img width="566" height="48" alt="image" src="https://github.com/user-attachments/assets/0ca6c25b-7534-4a05-8aee-f5d81ea8e5d4" />

Reply:
@sean you try the above suggestion:
>for OST, I recommend to initialize commerce using initService in ost.js
currently OST initializes mas-commerce-service if it doesn't find one already, so we need to make sure that mas-commerce-service is initialized by merch.js

Reply:
once the following line is reverted, I'm happy to approve.

> if (window.location.pathname.includes('/tools/ost')) commerce.allowOverride = 'true';